docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+auto-imported from a USB drive. See here and [system user][system-user] and
+[system user assertions.][system-user-assertions.] 
+
+The system-user assertion is signed by a key you create, register, export and
+upload to the Serial Vault just like the serial key. See the the content
+elsewhere in this doc for detailed instructions.
+ 
+The System-User menu lets you create system user assertion for a certain brand
+/model, as follows:
+1. Click System-User.
+1. Enter the email of the system user into Email field.
+1. Enter the desired login name of the system user in Username field.
+1. Enter the system user password in Password field.
+1. Enter complete name of the user in Full Name field.
+1. Select the model of the device this user is being created from Model combo
+box.
+1. Set date and time in UTC since this assertion is valid in Since (UTC)
+field.
+
+!!! Note:
+          This date must be after the date on which the key used to sign the
+          system user assertion was registered.  
+1. Press Create to generate the assertion.
+
+!['image'][serial-vault-image9]
+
+After pressing Create, the generated assertion is displayed. Click Download 
+to save it to a local file. 
+
+As explained in linked documents, you can place this file in the root
+directory of a USB drive and named auto-import.assert. If the USB drive is
+inserted in an unmanaged Core system, the assertion is imported and the system
+user is created. 
+
+## Verifying a system obtained a serial assertion
+
+You can verify from a terminal session on a system if a serial assertion has
+been retrieved from the Serial Vault in various ways.
+
+
+!!! Note:
+          If the Core system does not have a user so you cannot get a terminal
+          session into the device, you can create one with a system-user
+          assertion as explained elsewhere in this document.
+
+### Verifying serial with snap known serial
+
+Display the system’s serial assertion, if any, as follows:
+
+```bash
+snap known serial
+```
+
+The output should show a variety of information, including the serial identifier:
+
+```bash
+type: serial
+authority-id: canonical
+brand-id: mybrand
+model: rpi3
+serial: 381d1f99-749f-41f5-a667-608f698a4a4a
+device-key:
+	AcbBTQRWhcGAARAAzFqvP/235lVcpelVL2RxxHnDwi0viTLFdk3QxrSDfmLlTrP/griNH5zfDPHH
+Qf7mCC2yrjPLgSPmDvZm58Mnh3y+TFL9dF6lF6L0qXM865LO07sD1nZKE9lvl61jt+opmHviwTvb
+...
+```
+
+If there is no output from this command, the system does not have a serial assertion.
+
+
+
+### Checking Initialize device transaction with snap changes
+
+The system reports transactions with snap changes. The Initialize device
+transaction includes Serial Vault operations. If snap known serial does not
+output a serial assertion, you can troubleshoot further as follows:
+
+1. Display snap changes with the command:
+
+```bash
+snap changes
+```
+
+This will list the  transactions in reverse chronological order (most recent first), labelling each with an ID:
```

Comment:
Get rid of an extra space between "the  transactions".

Reply:
And delete "will". (don't use future tense). Instead: "This lists the..."

Reply:
done